content/zh/docs/tutorials/kubernetes-basics/cluster-intro.html

```diff
@@ -12,96 +8,118 @@
+                    <p><i>
+                        <!-- Kubernetes is a production-grade, open-source platform that orchestrates the placement (scheduling) and execution of application containers within and across computer clusters. -->
+                        Kubernetes 是一个生产级的开源平台，用于协调集群内部和跨集群的应用程序容器的承载（调度）和执行。
+                    </i></p>
+                </div>
+            </div>
+        </div>
+        <br>
+
+        <div class="row">
+            <div class="col-md-8">
+                <!-- <h2 style="color: #3771e3;">Cluster Diagram</h2> -->
+                <h2 style="color: #3771e3;">集群示意图</h2>
+            </div>
+        </div>
+
+        <div class="row">
+            <div class="col-md-8">
+                <p><img src="/docs/tutorials/kubernetes-basics/public/images/module_01_cluster.svg"></p>
+            </div>
+        </div>
+        <br>
+
+        <div class="row">
+            <div class="col-md-8">
+                <!-- <p><b>The Master is responsible for managing the cluster.</b> The master coordinates all activities in your cluster, such as scheduling applications, maintaining applications' desired state, scaling applications, and rolling out new updates.</p> -->
+                <p><b> Master 节点负责管理集群</b> master 负责协调集群中的所有活动，例如调度应用程序，维护应用程序的所需状态，扩展应用程序以及回滚更新。</p>
+                <!-- <p><b>A node is a VM or a physical computer that serves as a worker machine in a Kubernetes cluster.</b> Each node has a Kubelet, which is an agent for managing the node and communicating with the Kubernetes master. The node should also have tools for handling container operations, such as Docker or rkt. A Kubernetes cluster that handles production traffic should have a minimum of three nodes.</p> -->
+                <p><b> Node 节点是虚拟机或物理机，充当 Kubernetes 集群中的计算节点。</b>每个节点都有一个 Kubelet ，它是一个管理节点并与 Kubernetes master 节点通信的代理。该节点还应具有用于处理容器操作的工具，例如 Docker 或 rkt 。应用于生产的 Kubernetes 集群应至少有三个节点。</p>
+
+            </div>
+            <div class="col-md-4">
+                <div class="content__box content__box_fill">
+                    <!-- <p><i>Masters manage the cluster and the nodes are used to host the running applications.</i></p> -->
+                    <p><i> Master 节点管理集群，Node 节点用于承载运行的应用程序。</i></p>
+                </div>
+            </div>
+        </div>
+
+        <div class="row">
+            <div class="col-md-8">
+                <!-- <p>When you deploy applications on Kubernetes, you tell the master to start the application containers. The master schedules the containers to run on the cluster's nodes. <b>The nodes communicate with the master using the Kubernetes API</b>, which the master exposes. End users can also use the Kubernetes API directly to interact with the cluster.</p> -->
+                <p>在 Kubernetes 上部署应用程序时，您可以与 master 节点交互启动应用容器。 master 节点调度容器以在集群的 Node 节点上运行。 <b> node 节点使用 master 节点公开的 Kubernetes API </b>与 master 节点进行通信。最终用户还可以直接使用 Kubernetes API 与群集进行交互。</p>
```

Comment:
你可以 batch replace 一下 
master 节点 -> 主节点
node 节点 -> 工作节点
```suggestion
                <p>在 Kubernetes 上部署应用程序时，您可以与 master 节点交互启动应用容器。 master 节点调度容器以在集群的 Node 节点上运行。 <b> node 节点使用 master 节点公开的 Kubernetes API </b>与 master 节点进行通信。最终用户还可以直接使用 Kubernetes API 与群集进行交互。</p>
```